pkg/nordic_softdevice_ble/src/ble-mac.h

```diff
@@ -0,0 +1,50 @@
+#ifndef BLE_MAC_H
+#define BLE_MAC_H
+
+typedef enum {
+    BLE_EVENT_RX_DONE = 20000,
+} ble_mac_event_enum_t;
+
+#define BLE_IFACE_ADDED             (10000)
+
+#define BLE_SIXLOWPAN_MTU           (1280U)
+#define BLE_SIXLOWPAN_L2_ADDR_LEN   (8)
```

Comment:
According to @alexaring this is wrong, and should be 6. Apparently, it's wrong in Linux mainline kernel.